__tests__/endpoint.test.js

```diff
@@ -112,3 +124,31 @@ describe("GET: /api/articles", () => {
       });
   });
 });
+
+// Tests Task 6: -----------------
+describe("GET: /api/articles/:article_id/comments", () => {
+  test("Returns array of objects", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        expect(Array.isArray(response.body)).toBe(true);
+        expect(typeof response.body[0]).toBe("object");
+      });
+  });
+  test("Returned objects should include requested properties", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then((response) => {
+        response.body.forEach((object) => {
+          expect(object).toHaveProperty("comment_id");
+          expect(object).toHaveProperty("votes");
+          expect(object).toHaveProperty("created_at");
+          expect(object).toHaveProperty("author");
+          expect(object).toHaveProperty("body");
+          expect(object).toHaveProperty("article_id");
+        });
+      });
+  });
```

Comment:
Tests to add:
 [-] Status 400, invalid ID, e.g. string of "not-an-id"
 [-] Status 404, non existent ID, e.g. 0 or 9999
 [-] Status 200, valid ID, but has no comments responds with an empty array of comments